Week1/exercise/w1/index.js

```diff
@@ -28,3 +37,15 @@ console.log('header: ', header);
 //    create <li> element
 //    set <li> element innerHtml to todoInput value
 //    add <li> element to todoList
+
+const addTodoButton = document.querySelector("#btn-addTodo");
+const addTodo = () => {
+
+    const todoList = document.getElementById("todoList");
+    const todoInput = document.getElementById("todoInput");
```

Comment:
Idem, try selecting elements outside of the function, so that you don't select multiple times the same element